.make/dependencies/rmUnnecessaryFolders.php

```diff
@@ -52,22 +71,25 @@
 
 	try {
 		SetupUtils::rrmdir($sDir);
-		echo "OK Remove denied test dir: '$sDir'\n";
+		echo "✔️ Remove denied test dir: '$sDir'\n";
 	}
 	catch (\Exception $e) {
-		echo "\nFAILED to remove denied test dir: '$sDir'\n";
+		echo "\n❌ FAILED to remove denied test dir: '$sDir'\n";
 	}
 }
 
 
 $aAllowedAndDeniedDirs = array_merge(
-	$oiTopComposer->ListAllowedTestDir(),
-	$oiTopComposer->ListDeniedTestDir()
+	$oDependenciesHandler->ListAllowedQuestionnableFoldersAbsPaths(),
+	$oDependenciesHandler->ListDeniedQuestionnableFolderAbsPaths()
 );
-$aExistingDirs = $oiTopComposer->ListAllTestDir();
+$aExistingDirs = $oDependenciesHandler->ListAllQuestionnableFoldersAbsPaths();
 $aMissing = array_diff($aExistingDirs, $aAllowedAndDeniedDirs);
 if (false === empty($aMissing)) {
 	echo "Some new tests dirs exists !\n"
-		.'  They must be declared either in the allowed or denied list in '.iTopComposer::class." (see N°2651).\n"
-		.'  List of dirs:'."\n".var_export($aMissing, true);
-}
\ No newline at end of file
+		.'  They must be declared either in the allowed or denied list in '.$sDependenciesHandlerFQCN." (see N°2651).\n"
```

Comment:
Might be misleading to keep the old ticket reference as the code has evolved since then. 
Having both tickets references is too much also... 
I think simply having the message is enough ? 

```suggestion
		."  They must be declared either in the allowed or denied list in {$sDependenciesHandlerFQCN}\n"
```